src/main/java/Deadline.java

```diff
@@ -0,0 +1,62 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Listing {
+
+  private LocalDate deadLine;
+
+  /**
+   * Creates a new Deadline object by passing s as the Listing message and parsing deadline to
+   * LocalDate.
+   *
+   * @param s        detail of the listing
+   * @param deadline in the format YYYY-MM-DD
+   */
+  public Deadline(String s, String deadLine) {
+    super(s);
+    this.deadLine = LocalDate.parse(deadLine);
+  }
+
+  /**
+   * Creates a new Deadline object using an extra string parameter doneness. Used when creating
+   *    * Event objects are loading from storage
```

Comment:
Seems like a small typo with the second *?

Reply:
Noted and changed